pkg/kubelet/config/common.go

```diff
@@ -62,6 +63,7 @@ func applyDefaults(pod *api.Pod, source string, isFile bool, nodeName types.Node
 		if isFile {
 			fmt.Fprintf(hasher, "host:%s", nodeName)
 			fmt.Fprintf(hasher, "file:%s", source)
+			fmt.Fprintf(hasher, "time:%v", time.Now().Unix())
```

Comment:
doesn't this produce an unstable uid? does that mean every time the file is read the kubelet will delete/recreate the containers (including on kubelet restart if the containers were already running)?

Reply:
yep.. it does... need a different solution. I was thinking about using a md5 hash of the file, but this won't work either. The file contents are not guaranteed to change so the hash would bump.

Reply:
The last write time does work on windows... Switched to that.